src/test/ui/const-generics/issues/issue-74950.rs

```diff
@@ -0,0 +1,25 @@
+// [full] build-pass
+// revisions: full min
+#![cfg_attr(full, feature(const_generics))]
+#![cfg_attr(full, allow(incomplete_features))]
+#![cfg_attr(min, feature(min_const_generics))]
+
+
+#[derive(PartialEq, Eq)]
+struct Inner;
+
+// Note: We emit the error 5 times if we don't deduplicate:
```

Comment:
Can we deduplicate? This is very user-unfriendly at the moment.

Reply:
We deduplicate automatically on stable, the problem is that UI tests don't deduplicate by default